t/26-video_stream.t

```diff
@@ -106,6 +106,13 @@ subtest 'connect stream' => sub {
     is_deeply $cmd, [
         'ffmpeg', '-loglevel', 'fatal', '-i', '/dev/video0',
         '-vcodec', 'ppm', '-f', 'rawvideo', '-r', 3, '-'], "correct cmd built for fps=3";
+
+    $mock_console->redefine(update_framebuffer => sub ($self) { $self->{_last_update_received} = 200; return 0; });
+    $console->request_screen_update();
+    is $console->{dv_timings_supported}, 1, 'dv timings are supported';
+    ok $console->{dv_timings_last_check} >= time, 'dv_timings_last_check ok';
```

Comment:
There's https://perldoc.perl.org/Test::More#cmp_ok which gives a better error message in case the check fails.

The use of `>=` makes only sense when recording `time` before running the test, e.g. put `my $start_time = time;` before `$console->request_screen_update();` and compare against that. Otherwise, `time` might still be ahead of `dv_timings_last_check` if there's a big enough delay.

```suggestion
    ok $console->{dv_timings_last_check} >= time, 'dv_timings_last_check ok';
```

Reply:
I now see that my suggestion isn't changing anything. I guess I clicked on the button to add a suggestion but forgot to remove it after deciding not to go into concrete coding.

Can you just implement what I have written in text?

Reply:
I made the changes earlier itself https://github.com/os-autoinst/os-autoinst/pull/2626/files#diff-1320e7b8296971afbba67346b76e74f7b0cfbc5613a102f10406b36c9855beefR113

It's not visible here.

Reply:
Ah, right. But what I wrote in the 2nd paragraph hasn't been implemented yet. And if you think it doesn't make any sense what I wrote you can of course also say that :-)

Reply:
I have updated the code